test/e2e/common/empty_dir.go

```diff
@@ -205,6 +206,77 @@ var _ = Describe("[sig-storage] EmptyDir volumes", func() {
 	framework.ConformanceIt("should support (non-root,0777,default) [LinuxOnly] [NodeConformance]", func() {
 		doTest0777(f, testImageNonRootUid, v1.StorageMediumDefault)
 	})
+
+	It("pod should support shared volumes between containers", func() {
```

Comment:
Can we propose this for ConformanceIT  ?  BTW a nice follow on issue might be to add 'EmptyDir' to all these test descriptions.  This is a great test esp w/ diverging container runtimes, thanks !

Reply:
Let's make sure the test is stable first before promoting to conformance.

EmptyDir is already specified at the top level describe for these tests.